docs/docs/developers/getting_started/quickstart.md

```diff
@@ -2,86 +2,43 @@
 title: Quickstart
 ---
 
-In this guide, you will
+The easiest way to start developing on Aztec is simply to click on one of these buttons:
 
-1. Set up the Aztec sandbox (local development environment) locally
-2. Install the Aztec development kit
-3. Use Aztec.js to deploy an example contract that comes with the sandbox
-4. Use Aztec.js to interact with the contract you just deployed
+[![One-Click React Starter](/img/codespaces_badges/react_cta_badge.svg)](https://codespaces.new/AztecProtocol/aztec-packages?devcontainer_path=.devcontainer%2Freact%2Fdevcontainer.json) [![One-Click HTML/TS Starter](/img/codespaces_badges/vanilla_cta_badge.svg)](https://codespaces.new/AztecProtocol/aztec-packages?devcontainer_path=.devcontainer%2Fvanilla%2Fdevcontainer.json) [![One-Click Token Starter](/img/codespaces_badges/token_cta_badge.svg)](https://codespaces.new/AztecProtocol/aztec-packages?devcontainer_path=.devcontainer%2Ftoken%2Fdevcontainer.json)
 
-... in less than 10 minutes.
+That's it!
 
-## Prerequisites
+This creates a codespace with a prebuilt image containing one of the "Aztec Boxes" and a development network (sandbox). 
+- You can develop directly on the codespace, push it to a repo, make yourself at home.
+- You can also just use the sandbox that comes with it. The URL will be logged, you just need to use it as your `PXE_URL`.
 
-- Node.js >= v18 (recommend installing with [nvm](https://github.com/nvm-sh/nvm))
-
-## Install Docker
```

Comment:
Has this been added elsewhere? Or did you consider why this was fleshed out?

There were issues with Docker Desktop being open and how we use docker (on each command, vs as a running instance). There's some network check (at least) that occurs, so checking a version can take 5-30s. Other commands more (maybe multiple docker commands).
Basically if this needs to be refined or put elsewhere, please capture in an issue.

Reply:
Yes we should add this elsewhere, like in "common problems", I'll create an issue